pkg/apis/scheduling/types.go

```diff
@@ -64,6 +64,12 @@ type PriorityClass struct {
 	// when this priority class should be used.
 	// +optional
 	Description string
+
+	// NonPreempting specifies whether the pod which used this PriorityClass can start a preempting process.
```

Comment:
Given that a pod cannot start a preemption process, let's change the comment to:
```
// nonPreempting specifies whether a pod with this PriorityClass could trigger a preemption process.
// If this field is unset, the PriorityClass is considered a preempting class by default.
```